### ISSUE TITLE
Concardis\Payengine\Lib\Models\Response\Orders\Transaction::getOrder() can also return an array

### ISSUE THREAD

**Issue by ocean90** [concardis/PHP_SDK#4]
**Example 1:**
```php
$order_response = $payengine->orders( $order_id )->get();
$transactions = $order_response->getTransactions();
```

In this case `$transactions` is an array of `Concardis\Payengine\Lib\Models\Response\Orders\Transaction` objects. Using `$order->getTransactions()[0]->getOrder()` gets us the order ID as as a string as documented here: https://github.com/concardis/PHP_SDK/blob/71dd9b20cd8f0709272bfb86ad5bf17ec505524f/src/Concardis/Payengine/Lib/Models/Response/Orders/Transaction.php#L210-L216

**Example 2:**
```php
$transaction_response = $payengine->orders( $order_id )->transactions( $transaction_id )->get();
```

`$transaction_response` is again a `Concardis\Payengine\Lib\Models\Response\Orders\Transaction` object but now `$transaction_response->getOrder()` returns an array with all the properties of the associated `Concardis\Payengine\Lib\Models\Response\Order` model.

Is that behaviour by design? If so, should it be documented and/or why is `$order` in the second example not an instance of `Concardis\Payengine\Lib\Models\Response\Order`? Or should it always be the order ID as a string?

**Comment by Sharenkamp** [comment]
Hi @ocean90,

we will investigate this behavior. Thank for your feedback.

**Comment by Sharenkamp** [comment]
Fixed with 1.1.0

**Comment by ocean90** [comment]
Follow-up: #5
